src/pages/docs/administration/retention-policies/index.mdx

```diff
@@ -70,6 +70,13 @@ When a package retention policy is applied, Octopus will delete packages that me
 When configuring the repository retention policy, it's also worth making note of your [release retention policy](#releases-whats-deleted) settings too. When releases are deleted as a result of your release retention policy, then packages associated with those releases may become subject to cleanup by your repository policy.
 :::
 
+From Octopus Cloud version <INSERT_OCTOPUS_VERSION>, packages associated with releases in the Octopus Built-in repository can now be deleted. From the Built-in Repository Retention configuration page:
+
+- Select **A limited time** and configure the number of days to retain packages.
+- Select **Always keep packages used by releases and runbook runs on the dashboard**
+
+This option ensures that packages associated with releases and runbook runs are deleted from the Built-in Repository. Any packages associated with releases displayed on either the main dashboard or a [project dashboard](/docs/projects/project-dashboard) are **never deleted**. Any packages associated with runbooks displayed on the projects runbook dashboard are also **never deleted**.
```

Comment:
I think it's worth mentioning that the package publish time must also be exceeded, similar to the criteria above.